src/views/studio/modals/transfer-host-confirmation.jsx

```diff
@@ -0,0 +1,136 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import {FormattedMessage} from 'react-intl';
+
+import ModalInnerContent from '../../../components/modal/base/modal-inner-content.jsx';
+
+import TransferHostTile from './transfer-host-tile.jsx';
+import Form from '../../../components/forms/form.jsx';
+
+import {managers} from '../lib/redux-modules';
+
+import './transfer-host-modal.scss';
+
+const TransferHostConfirmation = ({
+    handleBack,
+    handleTransfer,
+    items,
+    hostId,
+    selectedId
+}) => {
+    const currentHostUsername = items.find(item => item.id === hostId).username;
+    const currentHostImage = items.find(item => item.id === hostId).profile.images['90x90'];
+    const newHostUsername = items.find(item => item.id === selectedId).username;
+    const newHostImage = items.find(item => item.id === selectedId).profile.images['90x90'];
```

Comment:
This is an extremely minor performance enhancement, but for example in studios with a large number of managers from before the update, this might be important, but wouldn't it be better to save the host and selected user objects and then get the properties instead of running `items.find` twice?

Like this:
```js
const currentHost = items.find(item => item.id === hostId);
const newHost = items.find(item => item.id === selectedId);
const currentHostUsername = currentHost.username;
const currentHostImage = currentHost.profile.images['90x90'];
const newHostUsername = newHost.username;
const newHostImage = newHost.profile.images['90x90'];
```